# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.cpp
# Commit: 2b880e8d9143
# Full Hash: 2b880e8d9143fee7badb103f965d461a4c1ceae8
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2020-07-03 15:35:32
# Regressor Bug: 1619523
# File Overlap Count: 1
# Description:
#   Bug 1619523 [Linux] Remove Wayland check at gfxPlatformGtk::UseHardwareVideoDecoding(), make VA-API decoding possible on X11 r=jhorak
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D82174
# ==============================================================================

diff -r 1b056b29e7fc -r 2b880e8d9143 gfx/thebes/gfxPlatformGtk.cpp
--- a/gfx/thebes/gfxPlatformGtk.cpp	Fri Jul 03 11:41:53 2020 +0000
+++ b/gfx/thebes/gfxPlatformGtk.cpp	Fri Jul 03 11:27:56 2020 +0000
@@ -732,7 +732,7 @@
           StaticPrefs::media_ffmpeg_vaapi_enabled());
 }
 bool gfxPlatformGtk::UseHardwareVideoDecoding() {
-  return IsWaylandDisplay() && gfxPlatform::CanUseHardwareVideoDecoding() &&
+  return gfxPlatform::CanUseHardwareVideoDecoding() &&
          StaticPrefs::media_ffmpeg_vaapi_enabled();
 }
 bool gfxPlatformGtk::UseDRMVAAPIDisplay() {
